src/emulator/gui/include/gui/functions.h

```diff
@@ -42,3 +43,17 @@ void draw_game_selector(HostState &host);
 void draw_reinstall_dialog(HostState &host, GenericDialogState *status);
 
 } // namespace gui
+
+// Extensions to ImGUi
+namespace ImGui {
+
+static auto vector_getter = [](void *vec, int idx, const char **out_text) {
```

Comment:
What's the point of this being a lambda.

Reply:
oh yea it was originally inside a func, forgot to make it a normal func when it was moved out